### DISCUSSION TITLE
Zigbee redmatic "cannot lock port"

### DISCUSSION THREAD

**Discussion by sebaskhh** [rdmtc/RedMatic#498] [General]
Hallo,
ich bekomme immer wieder die Meldung:
Status: Error while opening serialport 'Error: Error Resource temporarily unavailable Cannot lock port'

Setup:
- CCU3 mit CC253x -Zigbee stick
- redmatic 7.1.3 ist installiert, kein weiteres addon 
- (neueste Version habe ich auch schon probiert und noch mal "downgegraded)

Fehlerbeobachtung:
- mehrfache Neustarts  lösen das Problem manchmal. Allerdings steht irgendwann immer nicht "connected", sondern eben diese Fehlermeldung im Status.
- bei "ls /dev/tty*" ist "dev/ttyACM0" vorhanden. 
- da ich kein CUXd habe, kann ich kein "TTYASSIGN=ttyACM0:NC" eintragen. (das soll ja helfen..?)
- ich habe testweise nur dafür CUXd installiert, allerdings hat das nicht geholfen.
- ich habe den Stick testweise gegen einen conbee2 getauscht. Gleiches Problem.

Quellen:
https://github.com/hobbyquaker/node-red-contrib-zigbee/issues/21

Ich habe zwar tatsächlich Motivation, aber leider doch sehr begrenzt Ahnung von dem was ich da treibe... bitte um Verzeihung falls ich hier falsch gepostet habe.
Danke,
Sebas

**Comment by sebaskhh** [comment]
https://github.com/rdmtc/RedMatic/issues/499

toller Start ;) ich wollte das unter dem obigen Link unter "issues" posten und das hier löschen...

**Comment by hobbyquaker** [comment]
node-red-contrib-zigbee wird nicht mehr gepflegt und lässt sich auf RedMatic 9 nicht installieren (native serialport-Module). Empfehlung: zigbee2mqtt auf einem separaten Rechner und Anbindung per MQTT - siehe https://github.com/rdmtc/RedMatic/wiki/ZigBee. Geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
